packages/tupaia-web/src/features/Dashboard/DashboardItem/DashboardItem.tsx

```diff
@@ -0,0 +1,90 @@
+/**
```

Comment:
I'd be keen to move `DashboardItem` to the top level of features to save a level of nesting if you're keen?